assets/l10n/en-US.ftl

```diff
@@ -525,6 +525,10 @@ label_unread_messages = { $quantity ->
     *[other] {$quantity} unread messages
 }
 label_use_search = Use search to find an user
+label_user_added_to_group_by_link = {$addedUser} joined the group by invite link
+label_user_added_to_group_by_user = {$user} added {$addedUser}
+label_user_leave_group = {$removedUser} leave the group
+label_user_removed_from_group = {$user} removed {$removedUser}
```

Comment:
`label_user_added_to_group_by_link` -> `label_user_joined_group_by_link`
`{$addedUser} joined the group by invite link` -> `{$user} joined group by link`

`label_user_added_to_group_by_user` -> `label_user_added_user`
`{$user} added {$addedUser}` -> `{$author} added {$user}`

`label_user_leave_group` -> `label_user_left_group`
`{$removedUser} leave the group` -> `{$user} left the group`

`label_user_removed_from_group` -> `label_user_removed_user`
`{$user} removed {$removedUser}` -> `{$author} removed {$user}`